rllib/agents/a3c/a3c_tf_policy.py

```diff
@@ -129,4 +168,6 @@ def setup_mixins(policy, obs_space, action_space, config):
     postprocess_fn=postprocess_advantages,
     extra_action_fetches_fn=add_value_function_fetch,
     before_loss_init=setup_mixins,
-    mixins=[ValueNetworkMixin, LearningRateSchedule])
+    mixins=[ValueNetworkMixin, LearningRateSchedule],
+    view_requirements_fn=view_requirements_fn_pg,
```

Comment:
Can we call this `train_view_requirements_fn` for clarity?

Reply:
It's just that postprocessing view reqs are also in there, and they are often different from the train ones (e.g. next-obs for PPO).